src/reducers/linter.spec.tsx

```diff
@@ -501,6 +502,22 @@ describe(__filename, () => {
         actions.abortFetchLinterResult({ versionId }),
       );
     });
+
+    it('early returns and does not do anything when linter messages are already being fetched', async () => {
+      const url = '/some/url';
+      const versionId = 123;
+      const store = configureStore();
+      store.dispatch(actions.beginFetchLinterResult({ versionId }));
+
+      const { dispatch, thunk } = thunkTester({
+        createThunk: () => fetchLinterMessages({ url, versionId }),
```

Comment:
What about renaming `fetchLinterMessages` to `fetchLinterMessagesIfNeeded`? I think it would help. When looking at [this code](https://github.com/mozilla/addons-code-manager/blob/50be0327f1ae670d743a570855d23bef3eff14c3/src/components/LinterProvider/index.tsx#L77) it's not intuitive that the fetching action might *do nothing*. The name can provide a hint about what's going on.